modules/rte_mis_state/rte_mis_state.module

```diff
@@ -171,6 +169,7 @@ function rte_mis_state_form_alter(&$form, FormStateInterface $form_state, $form_
           if ($current_edit_user_id == $current_user_id) {
             $form['field_location_details']['widget'][0]['target_id']['#attributes']['readonly'] = 'readonly';
             $form['field_location_details']['widget'][0]['target_id']['#attributes']['disabled'] = 'disabled';
+            $form['account']['status']['#access'] = FALSE;
             // Update the roles options in the form, if the element is set.
```

Comment:
Please consider the case when the role parameter is missing or invalid for state/district admin.

Reply:
ok